pkg/ddevapp/ddevapp.go

```diff
@@ -2336,7 +2336,7 @@ func (app *DdevApp) Stop(removeData bool, createSnapshot bool) error {
 		util.Warning("Unable to SyncAndterminateMutagenSession: %v", err)
 	}
 
-	if globalconfig.DdevGlobalConfig.UseTraefik && status == SiteRunning {
+	if globalconfig.DdevGlobalConfig.Router == nodeps.TraefikRouter && status == SiteRunning {
```

Comment:
I'd introduce a method IsTrafik() or similar to the global config. Same for the proect config. Makes it much easier than using this comparision again and again.

Reply:
Done